library/borgbase.py

```diff
@@ -140,8 +176,16 @@ def main():
     else:
         key_id = get_key_id(module.params['ssh_key'])
 
-    # Add new repo using the key
-    res = add_repo(key_id)
+    # Check if repo with given name exist
+    repo_exist, repo_id = get_repo_id(module.params['repository_name'])
+
+    if repo_exist == True:
+        # Edit the repo
+        res = edit_repo(repo_id, key_id)
+
+    elif repo_exist == False:
```

Comment:
Just `else:`

Or is there a third case where variable `repo_exist` has a different value then true/false?